readthedocs/projects/tasks.py

```diff
@@ -592,6 +599,15 @@ def set_valid_clone(self):
         self.project.has_valid_clone = True
         self.version.project.has_valid_clone = True
 
+    def save_build_config(self):
+        """Save config in the build object."""
+        pk = self.build['id']
+        build_data = api_v2.build(pk).get()
```

Comment:
This feels a bit overkill. Can we not use the `patch` method to just update the config, without having to `get()` it previously? Similar to: https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/projects/tasks.py#L608